app/src/services/api/endpoints.ts

```diff
@@ -338,6 +341,17 @@ export const api = generatedApi
       apiBatchesDestroy: {
         invalidatesTags: invalidatesOne("Batches"),
       },
+      /**
+       * Pagai
+       */
+      apiExploreCreate: {
+        transformResponse: (response: ExplorationResponse) => ({
+          ...response,
+          rows: response.rows.map((row) =>
+            row.map((value) => value.toString())
```

Comment:
wouldn't be something that we should do when we display the "exploration table" ? In the component ?

Reply:
In this case we'd want to transform the response of a mutation and not a query. I've tried using the hook parameter `selectFromResult` but without success because this applies to the `data` value returned by the hook, and not `unwrap()`.

I've also directly called `onString` inside the returned JSX, but this is kind of ugly in my opinion...

I've come with this solution most of all because the exploration values are typed as string, so i've concluded that it would be best to parse them as string the soonest.

Reply:
Maybe that would do the trick

Reply:
@simonvadee @Jasopaum do you have some inputs on this?

Reply:
I'm not really in favor of transforming values to strings in the backend: we'd lose information about the types (which can be valuable in some cases). But I agree that the openAPI schema is misleading and should say "any". However I kind of agree with @tevariou on the fact that it could be done directly in the component (this logic is hidden now) or at least documented in the component.